scholly/base/src/global/fakeData.js

```diff
@@ -181,3 +181,16 @@ export const FAKE_TEAM_MEMBERS = [
 export const FAKE_TEAM_HOMEPAGE = {
   aboutUs: ['This is first paragraph.', 'This is second paragraph.', 'This is third paragraph.'],
 };
+
+export const FAKE_TEAM_ACHIEVEMENTS = [
+  {
+    title: 'Organisation of The Year',
+    description: 'We have been awarded as the Organisation of The Year for 2019',
+    yearAwarded: 2019,
+  },
+  {
+    title: 'Best Team Vibes',
+    description: 'We have been awarded for the Best Team Vibes for 2021',
+    yearAwarded: 2021,
+  },
```

Comment:
nice sample dataset here :rofl:

Reply:
just a nit: can we add a few more entries so we can test how the page handles more achievements than fits on the page, hence needing scrolling.

Reply:
Added more entries. The achievements page for the client website has not yet been implemented so can't tell how many entries for fake data need to be added to require scrolling, but this can be updated with more once the achievements page has been created.